src/draw/drawings.js

```diff
@@ -71,18 +74,18 @@ function createNonTraversableCellSprites(
  * the corresponding cells in green on the map.
  */
 export function drawPlannedPath(path, cpt, hexColor = 0x00ff00, alpha = 0.25) {
-  clearSprites(tagpro.renderer.pathSprites); // clear the previous path from the map
+  pathSprites = clearSprites(pathSprites); // clear the previous path from the map
```

Comment:
`pathSprites` stores a reference to the sprites that are currently draw on the tagpro map. Is that clear from implementation, or are additional comments necessary?